WordPress/Classes/Networking/BlogServiceRemoteXMLRPC.m

```diff
@@ -20,6 +20,25 @@ - (id)initWithApi:(WPXMLRPCClient *)api
     return self;
 }
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog success:(void(^)(BOOL isMultiAuthor))success failure:(void (^)(NSError *error))failure
```

Comment:
Break a bit so this line doesn't exceed 80 chars.

Also use the typedef'd block names for the parameters.